src/custom/hooks/useActivityDerivedState.ts

```diff
@@ -79,7 +79,7 @@ function getActivityDerivedState(props: {
   }
 }
 
-function getActivityLinkUrl(params: {
+export function getActivityLinkUrl(params: {
```

Comment:
If it's not an override of UNI-flies, let's move it to cow-react

Reply:
I think we should do that in another PR, this one is large enough.